scripts/sct_label_utils.py

```diff
@@ -598,6 +602,30 @@ def launch_sagittal_viewer(self, labels):
 
         return output
 
+    def remove_or_keep_labels(self, labels, action):
+        """
+        Create or remove labels from self.image_input
+        :param list(int): Labels to add or remove
+        :param str: 'remove': remove labels (i.e. set to zero), 'keep': keep labels
+        """
+        image_output = msct_image.zeros_like(self.image_input) if action == 'keep' else  self.image_input.copy()
+        coordinates_input = self.image_input.getNonZeroCoordinates()
+
+        for labelNumber in labels:
+            isInLabels = False
+            for coord in coordinates_input:
+                if labelNumber == coord.value:
+                    new_coord = coord
+                    isInLabels = True
+            if isInLabels:
+                if action == 'keep':
+                    image_output.data[int(new_coord.x), int(new_coord.y), int(new_coord.z)] = new_coord.value
+                else:
```

Comment:
replace "else" by "elif action == 'remove'" for more clarity and code expandability

Reply:
Understood. Changed in commit [cd29609](https://github.com/neuropoly/spinalcordtoolbox/commit/cd2960958c1f2a4215d14b7b53d21f5db8d9d096)